src/test/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentControllerIntegrationTest.java

```diff
@@ -76,7 +76,7 @@
 @ExtendWith(SystemStubsExtension.class)
 class SignDocumentControllerIntegrationTest {
 
-    private static final String LOCALSTACK_IMAGE_NAME = "localstack/localstack:4.0";
```

Comment:
For other reviewers - Localstack 1.4 is the latest version which works in the pipelines and as it happens the S3 api hasn't changed despite the latest Localstack being 4.x. Therefore fairly confident having an old version in itests is not going to be give us an invalid pass